spec.bs

```diff
@@ -1249,6 +1333,14 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
   1. Set |componentAuctionExpectedCurrency| to the result of [=looking up per-buyer currency=] with
     |topLevelAuctionConfig| and |auctionConfig|'s [=auction config/seller=].
 1. Let |pendingBuyers| be the [=map/size=] of |bidGenerators|.
+1. Set |additionalBids| to the result of running [=validate and convert additional bids=] with
+  |auctionConfig|, |topLevelAuctionConfig| and |negativeTargetInfo|.
+1. Set |pendingAdditionalBids| to the [=list/size=] of |additionalBids|.

```

Comment:
`|pendingAdditionalBids` and `|additionalBids|` are both _new_ variables right? Use `Let` instead of `Set`